src/cli/cli.cpp

```diff
@@ -6137,14 +6214,25 @@ template <> otError Interpreter::Process<Cmd("prefix")>(Arg aArgs[])
 #endif // OPENTHREAD_CONFIG_BORDER_ROUTER_ENABLE
 
 #if OPENTHREAD_FTD
-template <> otError Interpreter::Process<Cmd("preferrouterid")>(Arg aArgs[])
+template <> otError Cli::Interpreter::Process<Cli::Cmd("preferrouterid")>(Arg aArgs[])
 {
     return ProcessSet(aArgs, otThreadSetPreferredRouterId);
 }
 #endif
 
 #if OPENTHREAD_CONFIG_MAC_FILTER_ENABLE && OPENTHREAD_CONFIG_RADIO_LINK_IEEE_802_15_4_ENABLE
-template <> otError Interpreter::Process<Cmd("radiofilter")>(Arg aArgs[])
+template <> otError Cli::Interpreter::Process<Cli::Cmd("radiofilter")>(Arg aArgs[])
+
+/**
+ *@cli preferrouterid <routerid>
+ *@code
+ *preferrouterid 16
+ *Done
+ *@endcode
+ *@cparam preferrouterid {<routerid>}
+ *@par
+ *Specifies the preferred router ID that the leader should provide when solicited.
+ */
 {
     return ProcessEnableDisable(aArgs, otLinkIsRadioFilterEnabled, otLinkSetRadioFilterEnabled);
```

Comment:
These APIs should be `@sa` in the command i think 

otLinkIsRadioFilterEnabled
otLinkSetRadioFilterEnabled